lib/elixir/lib/file.ex

```diff
@@ -66,7 +66,7 @@ defmodule File do
   and `:delayed_write` are also useful when operating large files or
   working with files in tight loops.
 
-  Check http://www.erlang.org/doc/man/file.html#open-2 for more information
+  Check [`:file.open/2`](http://www.erlang.org/doc/man/file.html#open-2) for more information
```

Comment:
It could just be `:file.open/2` I think because we automatically link to the erlang docs too.